src/components/Form/Form.tsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import { useForm } from 'react-hook-form';
+import {
+  FormTextInput,
+  Button,
+  FormSelect,
+  Headline,
+} from '@vfde-react/pattern';
+import { DevTool } from '@hookform/devtools';
+import RootState from '../../types/rootState';
+import { setFormData, setUserType } from '../../actions';
+import { useDispatch } from 'react-redux';
+import { useNavigate } from 'react-router-dom';
+import './Form.css';
+
+const Form: React.FC = () => {
+  const navigate = useNavigate();
+  const dispatch = useDispatch();
+  const form = useForm<RootState>();
+  const { register, control, handleSubmit, formState } = form;
+  const { errors } = formState;
+  const onSubmit = (data: RootState) => {
+    dispatch(setFormData(data));
+    navigate('/overview');
+  };
+  return (
+    <>
+      <Headline
+        children="Form Page"
+        level={2}
+        mobileAlign="left"
+        align="left"
```

Comment:
no need for mobileAlign and align props

Reply:
this is because of the warnings appeared from the component i was trying to reduce the number of warnings by defining the data